src/ingress.ts

```diff
@@ -319,7 +338,7 @@ export class IngressBackend {
     });
   }
 
-  private constructor(private readonly backend: k8s.IngressBackend) {
+  constructor(private readonly backend: k8s.IngressBackend) {
```

Comment:
We intentionally didn't expose this constructor because of the factory methods that create it. cdk8s-plus intentionally hides inner kubernetes classes, sometimes that comes at the expense of flexibility, which we fix by extending the plus API. 

Can we support the `use-annotations` use-case by extending the `fromResource` and `fromService` methods? or maybe create a new factory method?